packages/generator/src/generator.ts

```diff
@@ -343,46 +381,67 @@ export abstract class Generator<
       try {
         templatedFile = this.prettier.format(templatedFile, options)
       } catch (error) {
-        console.warn(`Failed trying to run prettier: ` + error)
+        console.error(`Failed trying to run prettier: ` + error)
       }
     }
     return templatedFile
   }
 
+  async preFileWrite(filePath: string): Promise<any> {
```

Comment:
can we provide a better type compared to Promise<any>? is it a Promise<AppTemplateValues>?